src/components/content/order/services/Services.tsx

```diff
@@ -105,6 +106,9 @@ function Services(): React.JSX.Element {
                                                 <span className={serviceOrderStyles.serviceTypeOptionDescription}>
                                                     {item.content}
                                                 </span>
+                                                <span className={serviceOrderStyles.serviceTypeOptionVendor}>
```

Comment:
its better to place this at the right bottom corner of the service

Reply:
fixed